tools/export/exporters.py

```diff
@@ -159,3 +159,9 @@ def group_project_files(self, sources):
         """
         data = sorted(sources, key=self.make_key)
         return {k: list(g) for k,g in groupby(data, self.make_key)}
+
+    @abstractmethod
```

Comment:
ABCMeta is not the metaclass of Exporter. Without that, this `@abstractmethod` decorator is meaningless.